API/Controllers/BooksController.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using VL.Shared.Interfaces;
+using VL.Shared.Model;
+
+namespace API.Controllers
+{
+    [Authorize(AuthenticationSchemes = JwtBearerDefaults.AuthenticationScheme)]
+    [Route("api/vl")]
+    [ApiController]
+    public class BooksController : ControllerBase
+    {
+        private readonly IBookService _bookService;
+
+        public BooksController(IBookService bookService)
+        {
+            _bookService = bookService;
+        }
+
+        /// <summary>
+        /// Get all books
+        /// </summary>
+        /// <returns></returns>
+        [HttpGet, Authorize("IsAllUsers")]
+        [ProducesResponseType(typeof(bool), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(string), StatusCodes.Status400BadRequest)]
+        [ProducesDefaultResponseType]
+        public async Task<IActionResult> GetBooks()
+        {
+            var books = await _bookService.GetBooksAsync();
+            if (books == null) return BadRequest();
+            return Ok(books);
+        }
+
+        /// <summary>
+        /// Get a specific book by id
+        /// </summary>
+        /// <param name="id"></param>
```

Comment:
<param name="id"></param> is XML doc for saying there is a parameter

<param name="id">ID of the Book</param>

Reply:
Resolved - dont know why? 

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#returns

Reply:
I added info on all routes similar to the <returns> addition you demoed for me in the above comment: 
```
/// <summary>
/// Get all Books
/// </summary>
/// <returns>A list of books</returns>
```

I'm not sure what 'Resolved - dont know why?' means? Did I do this wrong?